src/components/TransactionItem/Details/Details.js

```diff
@@ -36,6 +36,9 @@ type Props = {|
 
 type StateProps = {|
   +isCommenting: boolean,
+  +isHoveredHash: boolean,
+  +isHoveredFrom: boolean,
+  +isHoveredTo: boolean,
 |}
```

Comment:
it's worth to add just one `hovered` prop with `'hash' | 'to' | 'from' | null` type. Therefore one `onHover` handler

Reply:
Agreed. Also less lag while moving pointer between different items.